event_dispatcher.rst

```diff
@@ -141,8 +141,8 @@ Creating an Event Subscriber
 
 Another way to listen to events is via an **event subscriber**, which is a class
 that defines one or more methods that listen to one or various events. The main
-difference with the event listeners is that subscribers always know which events
-they are listening to.
+difference with the event listeners is that subscribers always know the events
```

Comment:
the current wording sounds correct to me

Reply:
Ending a sentence with a preposition is acceptable in casual writing, but it is not advisable in formal writing.

Reply:
@gnito-org I merged your contribution ... but please keep in mind that even our docs are "formal", they must not be very formal.

Also, not related to this but, we must always use very basic vocabulary because English should not be a barrier for people wanting to learn Symfony (of course, readers/developers must have some level of English, but not too high).

Thanks!